src/types/models.ts

```diff
@@ -136,6 +136,7 @@ export interface ConnectWebview {
 
 export interface AccessCodeBase {
   access_code_id: string
+  device_id?: string
```

Comment:
I believe a returned access code should always include the device ID, so you could probably do 
```suggestion
  device_id: string
```

here.

Reply:
The return value in the tests exclude it:

```json
{
        access_code_id: '77541322-4c07-42c5-a1d0-0937428e4f35',
        code: '1234',
        created_at: '2022-09-28T21:45:05.537Z',
        name: 'new name',
        status: 'set',
        type: 'ongoing',
}
```